MCDataProducts/inc/KalSeedMC.hh

```diff
@@ -34,15 +34,17 @@ namespace mu2e {
     MCRelationship _rel; // relationship of this particle to its primary
     uint16_t _nhits; // number of associated StrawHits
     uint16_t _nactive; // number of associated active hits
-    XYZVectorF _mom; // initial momentum
+    XYZTVectorF   _mom; // initial momentum
```

Comment:
I may have mislead you; if you want this to be a Lorentz vector you should use
ROOT::Math::PxPyPzM4D<float>
instead.  That should be defined in a typedef.